src/main/resources/rhizome.yaml

```diff
@@ -1,6 +1,34 @@
-hikari:
-  jdbcUrl: jdbc:postgresql://localhost:5432/openlattice
-  username: "oltest"
-  password: "test"
-  maximumPoolSize: 98
-  connectionTimeout: 12000000
+##
+# Rhizome Bootstrap Configuration
+##
+
+# Set to false to disable persistence 
+enable-persistence:  false
+session-clustering-enabled: false
+
+hazelcast:
+    server: false
+    group: "local"
+    password: "trellis"
+    cp-member-count: 0
+    seed-nodes: 
+        - "localhost"
+
+hazelcast-clients:
+    IDS:
+        server: false
+        group: "local"
+        password: "trellis"
+        cp-member-count: 0
+        seed-nodes:
+            - "localhost"
+
+postgres:
+    citus: true
+    initialize-indices: true
+    hikari:
+        jdbcUrl: jdbc:postgresql://localhost:5433/openlattice
```

Comment:
this looks like the citus jdbcUrl -- is the intent that we should be running citus locally to run transporter?

Reply:
nope that's just what I'm developing against. this should be the same as the default for conductor, datastore, etc.